smartsnippets/tests/tests.py

```diff
@@ -12,6 +13,10 @@
     SmartSnippetPointer,
     Variable,
 )
+class FakeSiteAdmin(admin.ModelAdmin):
```

Comment:
I've added the explanation in the commit, but maybe it would be better to describe it in the code.